frontend_tests/node_tests/message_list_view.js

```diff
@@ -6,10 +6,26 @@ zrequire('XDate', 'node_modules/xdate/src/xdate');
 zrequire('Filter', 'js/filter');
 zrequire('FetchStatus', 'js/fetch_status');
 zrequire('MessageListData', 'js/message_list_data');
-zrequire('MessageListView', 'js/message_list_view');
-zrequire('message_list');
 
 var noop = function () {};
+var proxyquire =  require('proxyquire').noCallThru();
+// See https://github.com/tleunen/babel-plugin-module-resolver/issues/241
+// for why we need resolvePath
+var resolvePath = global.resolvePath;
+global.MessageListView = proxyquire('js/message_list_view.js', {
+    [resolvePath('js/rows')]: {
+        get_table: function () {
+            return {
+                children: function () {
+                    return {
+                        detach: noop,
+                    };
+                },
+            };
+        },
+    },
+}).default;
```

Comment:
Specifically this is the block that doesn't make sense to me test-wise.

Reply:
Is there a reason we can't just continue to mock `rows` with `set_global` here?

Reply:
See the comment below